pkg/kubernetes/client/kubectl.go

```diff
@@ -49,12 +53,32 @@ func (k Kubectl) Info() (*Info, error) {
 	}, nil
 }
 
+func (k Kubectl) ctl(action string, args ...string) *exec.Cmd {
+	// prepare the arguments
+	argv := []string{action,
+		"--context", k.context.Get("name").MustStr(),
+	}
+	argv = append(argv, args...)
+
+	// prepare the cmd
+	cmd := exec.Command("kubectl", argv...)
+
+	// add the nsPatch file to $KUBECONFIG
+	env := os.Environ()
+	for i, s := range env {
+		// TODO: handle empty $KUBECONFIG
+		if strings.HasPrefix(s, "KUBECONFIG=") {
+			env[i] = fmt.Sprintf("KUBECONFIG=%s:%s", k.nsPatch, strings.TrimPrefix(s, "KUBECONFIG="))
+		}
```

Comment:
Can you handle the lack of a KUBECONFIG before this gets merged? I'm not sure how many people define KUBECONFIG (I don't, for example).

Reply:
Look at the PR description :D

This is a `TODO` I'm kept for tomorrow 'cause to late to do it now :D